gerritlab/main.py

```diff
@@ -16,21 +16,21 @@
 from gerritlab.utils import Bcolors, msg_with_color, print_with_color, warn
 
 
-def merge_merge_requests(remote, local_branch):
+def merge_merge_requests(remote, final_branch):
     """Merges merge requests."""
 
     print("\nMerging merge requests:")
 
     # Get MRs created off of the given branch.
-    mrs = merge_request.get_all_merge_requests(remote, local_branch)
+    mrs = merge_request.get_all_merge_requests(remote, final_branch)
     if len(mrs) == 0:
-        print("No MRs found for this branch: {}".format(local_branch))
+        print("No MRs found for this branch: {}".format(final_branch))
```

Comment:
This change is a change in behavior (and maybe user expectation).

Before:
```
git checkout -b work
...
git lab
...
git lab -m
```

Would only merge the changes from the `work` branch. But now it tries to merge changes from *every* branch.

If we want to preserve that behavior we'd need the same info we have in `create_merge_requests`—the commits on the current branch and then `&` them with all merge requests (or something similar).

Reply:
Thanks for calling this out.  I'll work on a test case and fix.

Reply:
I think this is resolved now.